.github/workflows/graph-toolkit-released.yml

```diff
@@ -0,0 +1,44 @@
+name: Microsoft Graph Toolkit New Version Released
+on:
+  repository_dispatch:
+    types: [graph-toolkit-released]
+jobs:
+  notification:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: 14
+          
+      - name: Get Release Detail
+        id: detail
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const responseBody = "<p>Microsoft Graph Toolkit v" + "${{ github.event.client_payload.version }} has been released.\n Details can be found here: https://github.com/microsoftgraph/microsoft-graph-toolkit/releases/tag/v${{ github.event.client_payload.version }}";
+            return responseBody;
+
+      - name: Setup Email and Recipient 
+        id: email
+        run: |
+          emails="vsciotadt@microsoft.com;junjieli@microsoft.com"
+          subject="Microsoft Graph Toolkit v${{ github.event.client_payload.version }} Released"
```

Comment:
In the recipients, can you please include Zhidi, Ji and Yu as well?

Reply:
Sure, no problem~

Reply:
updated